example-android-kts/settings.gradle.kts

```diff
@@ -1 +1,15 @@
 include(":app")
+
+pluginManagement {
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.namespace == "com.btkelly") {
+                useModule("com.btkelly:gnag:" + requested.version)
+            }
```

Comment:
What's this doing?

Reply:
This allowed me to set the plugin name for the locally deployed plugin which was not resolving properly with the `plugins` block way of using plugins in the `.kts` build files.

Reply:
Neat!